spring-cloud-data-admin/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,14 @@
+spring:
+  application:
+    name: spring-cloud-data-admin
+  cloud:
+    config:
+      uri: http://localhost:8888
+#  cloud:
```

Comment:
should we have comments here describing the service discovery properties are an alternative to explicitly configuring the config server URL?